src/nccl_ofi_sendrecv.c

```diff
@@ -14,6 +14,7 @@
 
 #include <rdma/fabric.h>
 
+#include "nccl-headers/net.h"
```

Comment:
We shouldn't need this.  It's not wrong, but can you open a SIM so we can figure out why we need it?

Reply:
I did a few pr-ci runs without this commit and it passes, so it does appear that this was made redundant at some point. I'm pretty sure that this only manifested when `--enable-trace=yes` was passed. Would prefer to just land it at this point if that's OK with you.

Reply:
Yup, totally fine shipping as is (provided you open a SIM...)

Reply:
Opened 713. Thanks.